tests/cli/test_alice.py

```diff
@@ -24,7 +25,7 @@ def test_initialize_alice_defaults(click_runner, mocker):
     assert "nucypher alice run" in result.output
 
     # Auth
-    assert 'Enter NuCypher keyring password:' in result.output, 'WARNING: User was not prompted for password'
+    assert 'Enter NuCypher keyring password' in result.output, 'WARNING: User was not prompted for password'
```

Comment:
Did we specifically remove the colon, or did `click` change some internal functionality?

Reply:
We added the password length, moving the colon, but the premise of the assertion is weak anyways.